src/configure/templates/azurePipelineTemplates/dotnetcoreLinuxWebApp.yml

```diff
@@ -23,10 +23,10 @@ variables:
   buildConfiguration: 'Release'
 
   # Build Projects
-  buildProjects: "$(workingDirectory)/**/*.csproj"
+  buildProjects: "{{#equals}}{{{ workingDirectory }}} '.' true '**/*.csproj' '$(workingDirectory)/**/*.csproj'{{/equals}}"
```

Comment:
https://mseng.visualstudio.com/AzureDevOps/_workitems/edit/1733196/

Reply:
ahh yes, now I understand the issue. I could not get it from the build logs. Thanks :)